IntegrationTests.Common/Database/NorthwindSetup.cs

```diff
@@ -35,7 +36,7 @@ public static void SetupDatabase ()
       var connectionString = DatabaseConfiguration.ReplaceDataSource( connectionSettings.ConnectionString);
       var databaseAgent = new DatabaseAgent (connectionString);
 
-      var commandBatch = File.ReadAllText ("Database/Northwnd.sql");
+      var commandBatch = File.ReadAllText (Path.Combine(TestContext.CurrentContext.TestDirectory, "Database/Northwnd.sql"));
```

Comment:
This and some other manual changes have wrong formatting, please fix

Reply:
> 
> 
> I'm getting the following error with basically all VB tests:
> 
> ```
> OneTimeSetUp: SetUpAttribute attribute not allowed in a SetUpFixture
> ```
> 
> Please invest some time and try to fix that, asking Daniel is probably your best bet. Re-request once all tests are fine and all formatting fixes are applied. Thx

Id guess that since the rewriter cant handle VB files that these have to be migrated manually

Reply:
There's, unless I'm mistaken, just one SetupFixture in the entire test and no TestFixtureSetups so this should be Quick and Clean, even if the syntax looks a lot like Visual Basic.